examples/helloworld/src/main/scala/zio_grpc/examples/helloworld/HelloWorldClientMetadata.scala

```diff
@@ -86,12 +86,6 @@ object HelloWorldClientMetadata extends zio.App {
             .withMetadataM(userToMetadata(User("hello")))
             .sayHello(HelloRequest("World"))
         _ <- putStrLn(r1.message)
-
```

Comment:
This example is meant to demonstrate how to use safe metadata. Can this be replaced with safe code?

Reply:
I put a comment in the issue around the removal. assuming you are happy with the removal, then the lines above the removed code demonstrate how to use SafeMetadata in the userToMetadata method